src/clr-angular/utils/drag-and-drop/providers/drag-event-listener.service.ts

```diff
@@ -37,24 +42,27 @@ export class DragEventListenerService<T> {
     return this.dragEnd.asObservable();
   }
 
+  get dragStartPosition(): { pageX: number; pageY: number } {
+    return this.initialPosition;
+  }
+
   constructor(private ngZone: NgZone, private renderer: Renderer2, private eventBus: DragAndDropEventBusService<T>) {}
 
   private initialPosition: { pageX: number; pageY: number };
 
   // Draggable component sets these properties:
   public dragDataTransfer?: T;
   public group?: string | string[];
+  public dragStartDelay: number = 0;
```

Comment:
Just do `dragStartDelay = 0` since type is inferred. Do we want to have a default delay, say 100?